From 14bf88d0e60be791c96788842d6c8cbe405b4bd2 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Mon, 13 Feb 2017 17:35:30 +0800
Subject: [PATCH 1510/5242] MLK-13765 ARM: dts: imx7ulp-evk-sd1: remove
 property out for sd1 slot

commit  b610a333f71e9259f7472cd284ec78de935176a5 from
https://source.codeaurora.org/external/imx/linux-imx.git

The two properties pm-ignore-notify and keep-power-in-suspend need
to remove for the sd slot on base board(sd1 slot).

If not, after system suspend, once remove the card from sd1 slot,
then system can't resume successfully, resume process hung due to
dead lock.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk-sd1.dts |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
index 6cdb99a..a081809 100644
--- a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
@@ -1,5 +1,6 @@
 /*
  * Copyright 2016 Freescale Semiconductor, Inc.
+ * Copyright 2017 NXP
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -28,5 +29,7 @@
 	/delete-property/non-removable;
 	/delete-property/cd-post;
 	/delete-property/wifi-host;
+	/delete-property/pm-ignore-notify;
+	/delete-property/keep-power-in-suspend;
 	status = "okay";
 };
-- 
1.7.9.5

